Classes/WPEditorViewController.m

```diff
@@ -1062,18 +1062,19 @@ - (void)showInsertImageAlternatePicker
 
 - (void)recoverFromViewSizeChange
 {
+    // This hack forces the input accessory view to refresh itself and resize properly.
     if (self.isFirstSetupComplete) {
-        // Important: This is a complete and utter hack that compensates for the input accessory view
-        // not properly changing size classes (resizing) when the rest of the views in the editor VC do.
-        // Toggling the HTML button on the input bar quickly does not affect the view and forces the
-        // input accessory view (the format bar) to update itself. FWIW, setNeedsDisplay and
-        // setNeedsLayout do NOT work.
         if ([self.editorView isInVisualMode]) {
-            [self.editorView showHTMLSource];
-            [self.editorView showVisualEditor];
+            [[self.editorView focusedField] blur];
+            [[self.editorView focusedField] focus];
```

Comment:
I noticed that we where loosing the focus and selection on rotation, if we change the code like this it looks it works better: 
```
 WPEditorField *field = [self.editorView focusedField];
            [self.editorView saveSelection];
            [field blur];
            [field focus];
            [self.editorView restoreSelection];
```

Can you test on your iPad Pro to see if we don't mess up anything.